Makefile

```diff
@@ -452,7 +452,7 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	@echo Updating Dependencies
 
 	# Update all dependencies (does not update across major versions)
-	$(GO) get -u
+	$(GO) get -u ./...
```

Comment:
What's the precise change in behaviour here?

Reply:
The change makes it recurse into subdirectories to look for modules there - without it, go-get doesn't find any modules to update dependencies for. I suspect the behaviour change happened when we switched full to go modules about 6 months ago.